modules/internal/ChapelBase.chpl

```diff
@@ -2372,4 +2372,31 @@ module ChapelBase {
   inline proc _removed_cast(in x) {
     return x;
   }
+
+  /* The following two overloads support compiler-generated == and !=
+     operators for records with array fields */
```

Comment:
You might want to make this comment more general to encompass the other four, or add similar comments to those

Reply:
Good point, I've fixed it.